content/news/027/index.md

```diff
@@ -132,6 +132,24 @@ The developers have also written a blog post:
 [Soldat]: http://soldat.pl/
 [Rhai]: http://rhai.rs/
 
+### [Graph Game]
```

Comment:
I'm not sure this is the correct PR for this change ;-D

Reply:
This is just showing up cause I resolved conflicts, if you go to https://github.com/rust-gamedev/rust-gamedev.github.io/pull/825/files the diff looks right 😅